### ISSUE TITLE
bootloader instalation error

### ISSUE THREAD

**Issue by snopw** [calamares/calamares#1926]
version of calamares is 3.2.49.1 I tried installing artix linux KDE plasma https://iso.artixlinux.org/iso/artix-plasma-runit-20220123-x86_64.iso but the following error keeps happening The bootloader could not be installed. The installation command <pre>grub-install --target=x86_64-efi --efi-directory=/boot/efi --bootloader-id=Artix --force</pre> returned error code 1.

**Comment by adriaandegroot** [comment]
- BIOS + erase disk was fine. After reboot system comes up normally.
- EFI + erase disk was fine (as in: the install did not fail). After a reboot falls into EFI shell. It does install a grubx64.efi for Artix, but doesn't configure it properly. I tried to reboot the installer ISO and fix it from there, but grub-install complains about "failed to get canonical path of 'overlay'". I don't have enough EFI-EDK-shell-fu to boot up Artix from the EFI prompt.

Closing as "needs more information" -- I can't reproduce the problem you seem to have, not with the information given.
